extensions/short-io/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Short.io Changelog
+
+## [Initial Version] - {PR_MERGE_DATE}
+
+- Shorten links using your own domain via Short.io.
```

Comment:
Maybe you could also mention the other commands, at least the ability to search for shorten links

Reply:
Thanks for the suggestion, I changed the description to:
`Shorten and search links with your own domain via Short.io.`
How do you think?

Reply:
sounds good 👍